sql/search_derived/search_clients_last_seen_v1/init.sql

```diff
@@ -0,0 +1,81 @@
+CREATE TEMP FUNCTION
+  udf_12_zeroes() AS ([0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]);
+CREATE TEMP FUNCTION  udf_new_monthly_engine_searches_struct() AS (
+  STRUCT(
+    udf_12_zeroes() AS total_searches,
+    udf_12_zeroes() AS tagged_searches,
+    udf_12_zeroes() AS search_with_ads,
+    udf_12_zeroes() AS ad_click
+  )
+);
+--
+CREATE TABLE
+  search_clients_last_seen_v1
+PARTITION BY
+  submission_date
+CLUSTER BY
+  sample_id,
+  client_id
```

Comment:
I don't see any benefit to clustering on client_id. I'd be inclined to keep this consistent with historical ping tables and cluster on (channel, sample_id)

Reply:
It may improve join performance when joining this with `clients_daily` (_if_ the optimizer uses it).